jdaviz/configs/default/plugins/model_fitting/model_fitting.py

```diff
@@ -528,7 +528,10 @@ def _check_model_component_compat(self, axes=['x', 'y'], display_units=None):
         self._check_model_equation_invalid()
 
     def _on_global_display_unit_changed(self, msg):
-        axis = {'spectral': 'x', 'flux': 'y'}.get(msg.axis)
+        if 'spectral' in msg.axis:
+            axis = msg.axis.replace('spectral_', '')  # get 'x' or 'y'
```

Comment:
does this work?  Aren't the message axis values either `spectral` or `spectral_y` (not `spectral_x` or `spectral_y`)?

Reply:
whoops yeah, ill change that